src/update-all/index.ts

```diff
@@ -1,6 +1,7 @@
 const preload = () => import("./update-all-dialog");
 
 export const openUpdateAllDialog = () => {
-  preload();
-  document.body.append(document.createElement("esphome-update-all-dialog"));
+  preload().then(() => {
```

Comment:
preload is a Promise, so I think we should use then or await it. Please confirm, so I'll do the same in other places

Reply:
it's a promise but you don't need to await it. Elements are automatically upgraded when loaded.

Reply:
So I should remove `then` or can we leave it as is?

Reply:
revert the change.

Reply:
done